### PR TITLE
Add Windows installation guide

### PR DESCRIPTION
**Pull request by Unrooted**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +190,51 @@ This will install the `termonad` binary to `~/.local/bin/`.
...
+
+* any X server app, for example **Vcxsrv**
+* any WSL, for example **Ubuntu**
+
+I'm using both Vcxsrv and Ubuntu WSL.
+
+Configure both Vcxsrv and WSL. For Vcxsrv go with default settings everywhere, it will be fine. Configure your WSL as you want (choose your name etc.). After you set up the user, you'll have to update your OS, run:
+
+```sh
+$ sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get upgrade -y && sudo apt-get dist-upgrade -y && sudo apt-get autoremove -y
```
Comment (cdepillabout) [inline README.md:203]:
```suggestion
$ sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get dist-upgrade -y && sudo apt-get autoremove -y
```

**Comment by cdepillabout** [comment]
@Unrooted I made a couple small changes to the Windows instructions.  Could you do a review of this?  If it looks alright to you, I'll go ahead and merge this.

(I also changed the `stack.yaml` file so it is no longer a soft-link.  This should simplify the instructions for windows slightly.)

**Comment by Unrooted** [comment]
It's better after your changes. Checked again everything and everything works fine.

**Comment by cdepillabout** [comment]
Thanks for taking the time to send this PR.  It should be a big help for people wanting to run this on Windows.

**Pull request merged**